src/platforms/ios/ioslogger.swift

```diff
@@ -102,6 +134,31 @@ public class IOSLoggerImpl : NSObject {
             logger.error(message: "Unable to access log file at \(appexLogFileURL): \(error)")
         }
     }
+
+    private static func withAppSwiftLogFile(_ f: (_ handle: FileHandle) throws -> Void) {
```

Comment:
nit: The only difference between this function and `withAppexLogFile` is the state of the empty file. I think we can merge these two functions into one that gets the data from an empty log file as a string.